.github/workflows/check.yml

```diff
@@ -116,22 +111,21 @@ jobs:
       - name: Run client/server transfer
         run: |
           cargo +${{ matrix.rust-toolchain }} build $BUILD_TYPE --bin neqo-client --bin neqo-server
-          cargo +${{ matrix.rust-toolchain }} run $BUILD_TYPE --bin neqo-server -- $HOST:4433 &
+          target/$BUILD_DIR/neqo-server $HOST:4433 &
           PID=$!
-          # Give the server time to start.
-          sleep 3
```

Comment:
Why remove the sleep? If I understand the failure below, the client is starting up faster than the server.

https://github.com/mozilla/neqo/actions/runs/8374577016/job/22930136500?pr=1692